v2/en/linux-packages.md

```diff
@@ -298,20 +298,29 @@ sudo apt-get -y install --no-install-recommends openresty
 See the [OpenResty Deb Packages](deb-packages.html) page for more details on all available packages in this
 repository.
 
-# CentOS
+# CentOS 8
 
 You can add the `openresty` repository to your CentOS system so as to easily install
 our packages and receive updates in the future (via the `yum update` command). To add the repository, just
 run the following commands (replace `yum` with `dnf` below if you are using CentOS 8+):
 
+- CentOS 8 or older
```

Comment:
We should put later versions before older versions? Also, should use `## xxx` etc, as mentioned above.